code/modules/gear_presets/uscm.dm

```diff
@@ -52,6 +56,17 @@
 			var/obj/item/device/radio/headset/almayer/marine/equipped_headset = H.wear_r_ear
 			equipped_headset.add_hud_tracker(H)
 
+/datum/equipment_preset/uscm/load_rank(mob/living/carbon/human/Player)
+	//Is it highpop, and does it have either a highpop or playtime rank?
+	if(GLOB.ishighpop && (highpop_paygrade || playtime_rank))
+		if(!playtime_rank)
+			return highpop_paygrade
+
+		//Specs & SGs get their playtime rank on highpop
+		if(get_job_playtime(Player.client, rank) < JOB_PLAYTIME_TIER_1)
```

Comment:
Shouldn't this be >= rather than <

Reply:
I just looked at the one that changes the Marine grunt rank. Will change